docs/backend/testing/running-server-tests-inside-a-transaction.md

```diff
@@ -4,57 +4,199 @@ sidebar_position: 9
-3. Remove any calls to `suite.TruncateAll()` in the package's tests.
-3. In the function that defines the test suite (it's usually the last function, and its name starts with `Test` and ends with `Suite`, such as `TestHandlerSuite`), add `, testingsuite.WithPerTestTransaction()` after `testingsuite.CurrentPackage()`. For example:
+**New Pattern**:
 
 ```golang
-hs := &HandlerSuite{
-    BaseHandlerTestSuite: handlers.NewBaseHandlerTestSuite(
-        logger,
-        notifications.NewStubNotificationSender("adminlocal", logger),
-        testingsuite.CurrentPackage(),
-        testingsuite.WithPerTestTransaction(),
-    ),
-}
+func (suite *MTOShipmentServiceSuite) TestMTOShipmentUpdater() {
+
+    // TEST SETUP
+    // Set up mocked objects and other local (non-db) variables
+    
+    // MOVE DB SETUP TO A FUNCTION
+    setupTestData := func() {  
+        // Prepare the database with testdatagen and suite.DB() calls
+    }
+
+    suite.Run("Subtest 1", func() {
+        setupTestData()    // <-- Call the DB setup function
+        // Run Test
+        // Check errors
+    }
+
+    suite.Run("Subtest 2", func() {
+        setupTestData()  
+        // Run Test
+        // Check errors
+    }
+
 ```
-Here's an [example commit that converts the ghcapi package](https://github.com/transcom/mymove/pull/6650/commits/ab1e72fbd559b73dc7a9089c0d5d5d12d4f83ba2).
 
-4. Run the tests and see if any fail. If none fail, then you are done! If tests fail, read on to fix them.
+### What's different
+
+:::note View in PR
+You can see these [changes in a PR](https://github.com/transcom/mymove/pull/6650/commits/3baaff02ffa8ca745ffb9c75422a1598180635ec#diff-a9eeef9ef97657c461ec7c76226f6c6f5dd03b4938217e4ed0de9bbbfe161815) if you prefer.
+:::
+
+Follow these directions
+
+1. The function declaration for the main test needs to be simplified.
+
+   * `suite.T().Run` → `suite.Run`
+   * `func(t *testing.T)` → `func()`
+
+   From:
+
+   ```golang
+    suite.T().Run("Subtest 1", func(t *testing.T) {
+   ```
+
+   To:
+
+   ```golang
+    suite.Run("Subtest 1", func() {
+   ```
+
+2. All DB setup should be moved to a function that is called in each subtest. There are more than one ways of achieving this, the following is just one example.
+
+   ```golang
+    // MOVE DB SETUP TO A FUNCTION
+    setupTestData := func() {  
+        // Prepare the database with testdatagen and suite.DB() calls
+    }
+
+    suite.Run("Subtest 1", func() {
+        setupTestData()    // <-- Call the DB setup function
+        // Run Test
+        // Check errors
+    }
+   ```
+
+3. Remove any calls to `suite.TruncateAll()` in the tests.
+4. Replace `suite.T().Fatalf` with `suite.Fail`.
+5. Remove `testing` from the imports at the top of the file.
+6. Run all tests in the package.
+
+## How to update the package to the new pattern
+
+Hopefully the package  you are working on has already been updated to use the new pattern.
+
+### **Check if packate has been updated**
```

Comment:
Small typo here - "packate" instead of "package". Sad that Vale didn't catch this too - does it not catch regular spelling errors? Hmm...